.github/dependabot.yml

```diff
@@ -0,0 +1,16 @@
+version: 2
+updates:
+- package-ecosystem: bundler
+  directory: "/"
+  schedule:
+    interval: daily
+    time: "03:00"
+    timezone: Europe/London
+  open-pull-requests-limit: 10
+  reviewers:
+  - mpw5
+  - jrmhaig
+  - naseberry
+  - imtiazAhmd
+  - NenehB
```

Comment:
I think all these users are in the `ministryofjustice/laa-claim-for-payment` team anyway, so I'm not sure they need listing separately - line 16 should do everything we need. 

Not a big issue but it means that if anyone leaves the team we would need to make an extra change to this file to remove them.

Reply:
Good point. I'll remove them.